modules/tinymce/src/core/main/ts/api/ui/Registry.ts

```diff
@@ -41,9 +41,9 @@ const registry = () => {
      * Registers a new contextual form item.
      * Similar to a context menu item, a contextual form is an item with an input
      * form element appearing when a content predicate is matched. An example
-     * us of a contextual form is the link plugin when the configuration
+     * of a contextual form is the link plugin when the configuration
      * { link_context_toolbar: true } is used. When the cursor is on a link, a
-     * contextual input form appears allowing for quick changes to the url field.
+     * contextual input form appears allowing for quick changes to the url field. For information on creating context forms, see: <a href="https://www.tiny.cloud/docs/ui-components/contextform/">UI Components - Context forms</a>.
```

Comment:
Break this and the others up over multiple lines please, as this line is far too long.

Reply:
fixed by https://github.com/tinymce/tinymce/pull/5730/commits/ea56e84f2f4ddd1ddb3fb208f21e524d6d80a279